uiowa/uiowa
# [IWP] Media Gallery

Resolves #7794 





# How to test



```
ddev blt ds --site=iwp.uiowa.edu && ddev drush @iwp.local uli /admin/structure/taxonomy/manage/writer_bio_countries/add
```
Add some countries
Also add some terms to the Genre taxonomy

Add a few _Resource_ nodes
Add a _Site Custom::Resource List_ block to a _Page_
Test the filters (note: the resources should appear as cards and pull the _Featured image_ as the card image).
The Search filter should search against the title, body, summary, and year (important, as searching against the year is required in order to not create a separate filter for it)

Check the content type creation best practices a la https://iowa.sharepoint.com/sites/drupal-developer-documentation/SitePages/Miscellaneous/Content%20type%20creation%20notes/Content%20type%20creation%20notes.aspx

Displays are not themed at this point, and will be themed as part of a later slice.
